pyodide-build/pyodide_build/tests/test_pypi.py

```diff
@@ -143,9 +143,15 @@ def fake_pypi_server():
         simple_root = root / "simple"
         if not simple_root.exists():
             simple_root.mkdir(exist_ok=True, parents=True)
-            # top package resolves_package that should resolve okay
+            # top package resolves_package that should resolve okay - nb:
+            # this package depends on bcrypt which is in pyodide already
+            # and should not be rebuilt
             _make_fake_package(
-                simple_root, "resolves-package", "1.0.0", ["pkg-a", "pkg-b[docs]"], True
+                simple_root,
+                "resolves-package",
+                "1.0.0",
+                ["pkg-a", "pkg-b[docs]", "bcrypt"],
```

Comment:
Maybe let's use `micropip` instead of `bcrypt`, which we guarantee that it will always be in the pyodide-lock.json.

Reply:
have done